addons/website_forum/controllers/website_forum.py

```diff
@@ -379,18 +377,21 @@ def post_create(self, forum, post_parent=None, **post):
             'parent_id': post_parent and post_parent.id or False,
             'tag_ids': post_tag_ids
         })
+        if post_parent:
+            post_parent._update_last_activity()
         return request.redirect("/forum/%s/%s" % (slug(forum), post_parent and slug(post_parent) or new_question.id))
 
     @http.route('/forum/<model("forum.forum"):forum>/post/<model("forum.post"):post>/comment', type='http', auth="user", methods=['POST'], website=True)
     def post_comment(self, forum, post, **kwargs):
-        question = post.parent_id if post.parent_id else post
+        question = post.parent_id or post
```

Comment:
Would say keep the if-else construct, even my linter yells on me for that (especially this is useless diff)